### DISCUSSION TITLE
Lat-Lon Grid and Equal-Area Projection

### DISCUSSION THREAD

**Discussion by ecemek** [mhm-ufz/mHM#66] [Q&A] [answered]
Dear mHM Team,

I'm a fresh user of the mHM so I have some hesitations about the data preparation before running the model. I would like to ask some questions.

1- My meteorological data is on lat-lon grid. Let's assume that the data's longitude range is 40 to 42 while lat range is 30 to 32.5 when checked with 'cdo sinfo'. So on the header.txt file I should put 40 for "xllcorner" and 30 "yllcorner" right? (iFlag_cordinate_sys = 1)

2- This is what confuses me the most. It is suggested on the documentation (Data Preparation for mHM page) that I put my meteorological data (.nc file) to the GIS system in order to prepare other datasets with the same extents. When I add the .nc file as a raster, its extents are changed and become something like 40.125 instead of 40 and 32.62 instead of 32.5 for example. I solve that by manually entering the true extents. On GIS I use WGS-84 EPSG:4326 and it has decimal degrees (lat-lon) as unit so my meteorological data is georeferenced correctly (when true extents are given manually). However it is strongly suggested on the documentation that we use an equal-area projection. As far as I know most of them if not all of them has units of meters. So wouldn't using them might cause some irregularities on the extents of my coarser domain. Or should I do all of the operations on GIS by using WGS84 then reproject all finished datasets to an equal-area one?  What are some tips or the most efficient ways to overcome this problem? If I use (iFlag_cordinate_sys = 0), how can I determine the xllcorner and yllcorner of my meteorological data if its on lat-lon grid? 

3- It is said on the documentation (Data Preparation for mHM page) that on .nc files every variable should be in DOUBLE format except "time" variable which should be INTEGER. I understand that variables like pre,tavg should be double; lat and lon should be double as well? Also, in the meteo files of the official test domain, "time" variable is DOUBLE instead of INT. 

Thanks in advance.

**Comment by rohinikumar82** [comment, marked as answer]
Answer for

1: Yes

2: The documentation is possibly not updated - thank you for noticing that - and mHM supports both the regular lat/lon (WGS84) and projected coordinated system (e.g., regular meter). For the meteorological forcings - since this is a huge database - we do not use ArcGIS for processing. Maybe you can use other programs like R or Python to create consistent netcdf forcing files for mHM.

3.  Yes, Lat/Lon can be in double.

**Reply by ecemek** [reply]
Thanks for the answer.

**Comment by rohinikumar82** [comment]
Answers for your queries: 

1: Yes

2: The documentation is possibly not updated - thank you for noticing that - and mHM supports both the regular lat/lon (WGS84) and projected coordinated system (e.g., regular meter). For the meteorological forcings - since this is a huge database - we do not use ArcGIS for processing. Maybe you can use other programs like R or Python to create consistent netcdf forcing files for mHM.

3.  Yes, Lat/Lon can be in double.

**Comment by rohinikumar82** [comment]
Answers for your queries: 

1: Yes

2: The documentation is possibly not updated - thank you for noticing that - and mHM supports both the regular lat/lon (WGS84) and projected coordinated system (e.g., regular meter). For the meteorological forcings - since this is a huge database - we do not use ArcGIS for processing. Maybe you can use other programs like R or Python to create consistent netcdf forcing files for mHM.

3.  Yes, Lat/Lon can be in double.
